msal/src/Microsoft.Identity.Client.Ref/Microsoft.Identity.Client.Ref.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <!-- This project is used to generated a ref assembly for MSAL. Use #if REFERENCE_ASSEMBLY in the code files. 
+  See https://oren.codes/2018/07/09/create-and-pack-reference-assemblies-made-easy/ for details-->
+  <PropertyGroup>
+    <TargetFrameworks>net45;netstandard1.3;uap10.0;MonoAndroid8.1;Xamarin.iOS10;netcoreapp1.0</TargetFrameworks>
```

Comment:
So I am adding conditional targets for the macOS build in the ADAL/MSAL/CORE csproj files. With these changes, do I need to also modify the new Ref.csproj files for the macOS build

Reply:
the targets in the ref do not need to match the implementation, but the public surface area should be there that you want the compiler to see on each platform.

Reply:
Thanks Oren. Discussed this offline with Travis - for the build on Mac (which we need for test purposes) we need to use a subset of the target frameworks. Travis is right - the simplest will be to use the same targets (on Win and on Mac) on both MSAL and MSAL.ref csproj files.